research-hub-web/src/app/components/cards/org-unit-card/org-unit-card.component.html

```diff
@@ -4,7 +4,7 @@
   <div style="margin-top: 12px"></div>
 
   <mat-card-title>
-    <a class="card-title" [attr.href]="orgUnit.link ? orgUnit.link : null">
+    <a class="card-title" tabindex="0" (keydown.enter)="orgUnit.link ? navigateTo(orgUnit.link) : null">
```

Comment:
So here the title is clickable but does no action?

Reply:
The whole card is clickable, so the title doesn't need to have the link - I realise this seems kind of odd, but the problem is that if the title does have the link then you get a new tab opening the link as well as the hub navigating to the link (double navigation). The keydown.enter is needed to navigate when the title is focused.

Reply:
Alternative is that the external links open in the current tab, but this feels a bit weird to me? What do you think? I feel like we should keep the user on the ResearchHub and open links in new tabs. But maybe that's just my personal preference making me biased :)

Reply:
@rosemcc An alternative, and possibly better, way to handle accessibility would be to make tabbing focus the whole card and then let aria-label read out the title of the card. Therefore the titles wouldn't need to be anchors any longer

This works fine though